packages/webpack/src/config/base.js

```diff
@@ -127,6 +126,18 @@ export default class WebpackBaseConfig {
     }
   }
 
+  cache() {
+    if (this.options.build.cache) {
```

Comment:
Maybe it's worth inverting this condition here and use an early return? :)

Reply:
Yes, that will be better